operators/operator-reference.adoc

```diff
@@ -36,6 +36,12 @@ include::modules/cluster-network-operator.adoc[leveloffset=+1]
 include::modules/cluster-samples-operator.adoc[leveloffset=+1]
 include::modules/cluster-storage-operator.adoc[leveloffset=+1]
 include::modules/cluster-version-operator.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
```suggestion
== Additional resources
```
General comment for the assembly. Per our Jupiter guidance [1] the header for Additional resources should start with `==` in an assembly. If you have time, please correct other instances of this while you are here. Thanks!
[1] https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=JUP&title=Jupiter+Source+files+readiness+Checklist#JupiterSourcefilesreadinessChecklist-Additionalresourcesspecificrequirements